### DISCUSSION TITLE
请问如何在自定义词典｢custom_phrase.txt｣里追加包含｢Tab /换行｣的词组?

### DISCUSSION THREAD

**Discussion by nutnutwin** [rime/home#958] [Ideas]
我知道可以在
custom_phrase.txt 里追加
> abc<A_Tab>def<A_Tab>255

这样输入def可以得到abc

请能能否在变换后的｢abc｣里加入Tab? 

比如输入def得到｢abc<A_Tab>｣

类似的, 能否在变换后的词组里加入换行? 

谢谢

**Comment by MokOopsing** [comment]
\n 或 \t 不知道是否可以，未测试

**Reply by nutnutwin** [reply]
> \n 或 \t 不知道是否可以，未测试

 \tabc	abc	255
这样不行...

**Comment by lotem** [comment]
不支持的。
超綱了。
有什麼具體使用場景？

**Comment by nutnutwin** [comment]
@lotem 
好的. 我考虑的情况是回复邮件时候直接插入文字段, 比如

```
你好, 我是某公司某部门的某某
感谢你的来电

mailto: xxx
```

我现在是用Snippet (AutoHotkey )来实现, 如果能用输入法的话, 某些不能用AutoHotkey 的场景会比较方便, 而且也方便管理
(AutoHotkey Script 不如词典方便

谢谢

**Comment by LEOYoon-Tsaw** [comment]
換行可以，tab不可

**Reply by nutnutwin** [reply]
> 換行可以，tab不可

请问应该怎样写custom_phrase.txt 呢? 
> \nabc	abc	355

这样好像不行, 是要用什么特殊符号吗谢谢
